Dockerfile

```diff
@@ -17,6 +17,9 @@ RUN cp /etc/ironic/ironic.conf /etc/ironic/ironic.conf_orig && \
     crudini --set /etc/ironic/ironic.conf DEFAULT debug true && \
     crudini --set /etc/ironic/ironic.conf DEFAULT default_network_interface noop && \
     crudini --set /etc/ironic/ironic.conf DEFAULT enabled_boot_interfaces pxe,ipxe && \
+    crudini --set /etc/ironic/ironic.conf DEFAULT enabled_power_interfaces ipmitool,idrac && \
+    crudini --set /etc/ironic/ironic.conf DEFAULT enabled_management_interfaces ipmitool,idrac && \
+    crudini --set /etc/ironic/ironic.conf DEFAULT enabled_hardware_types ipmi,idrac && \
     crudini --set /etc/ironic/ironic.conf DEFAULT default_boot_interface ipxe && \
     crudini --set /etc/ironic/ironic.conf DEFAULT default_deploy_interface direct && \
     crudini --set /etc/ironic/ironic.conf DEFAULT enabled_inspect_interfaces inspector && \
```

Comment:
You may want to enable their OOB inspection as well.

Reply:
Looks like the CI didn't test this, I'll update it once the CI fix merges
https://github.com/metalkube/dev-scripts/pull/156